doc/api/configuration.md

```diff
@@ -240,4 +240,23 @@ print(res)
 %config SqlMagic.style = "SINGLE_BORDER"
 res = %sql SELECT * FROM languages LIMIT 2
 print(res)
-```
\ No newline at end of file
+```
+
+## `persist_snippets`
+
+Default: `False`
+
+Stores the snippets as SQL files
+
+```{code-cell} ipython3
+%config SqlMagic.persist_snippets = True
+```
+
+```{code-cell} ipython3
+%%sql --save rated_high
+select * from languages where rating > 10.5
```

Comment:
can we hide it from the docs?

<img width="821" alt="image" src="https://github.com/ploomber/jupysql/assets/103026689/ab827e07-2cf1-48b8-8a85-fc8bba759ba3">

Reply:
This is a note for the users, to not edit the .sql files manually, it is displayed whenever we save the snippet. Not sure how to hide it just in the docs. Any pointers?

Reply:
> This is a note for the users, to not edit the .sql files manually, it is displayed whenever we save the snippet. Not sure how to hide it just in the docs. Any pointers?

Ok, let's keep it the message. However, the message looks like part of the article:

![image](https://github.com/ploomber/jupysql/assets/103026689/420dad19-b78c-439f-85b8-2f5831047b3e)

can we use the style we are already using? for example, make it italic or smaller, like one of these

![image](https://github.com/ploomber/jupysql/assets/103026689/e240b580-d6b1-4ed9-b167-3bb0dc74b15f)

Reply:
I have added styling to the message, I made a small change to our display.message method wherein now we can pass style as an attribute to the message 
<img width="937" alt="Screenshot 2023-07-26 at 6 08 13 PM" src="https://github.com/ploomber/jupysql/assets/69951190/ef541686-b11d-48dd-9907-05521171f3d8">

Reply:
I don't see the style [in our docs](https://jupysql--741.org.readthedocs.build/en/741/api/configuration.html)